### ISSUE TITLE
At OARC36 Petr Spacek presented a wide spread NSEC bitmap error

### ISSUE THREAD

**Issue by ulrichwisser** [zonemaster/zonemaster#1018]
At OARC36 Petr Spacek presented a widespread NSEC / NSEC3 bitmap error.
DNSVIZ has an option to test for this.
I believe Zonemaster should implement this too.

**Comment by mattias-p** [comment]
Here's a link: https://indico.dns-oarc.net/event/40/contributions/899/

That page links to a PDF with slides that I'm attaching here for good measure: [nsec-bitmaps.pdf](https://github.com/zonemaster/zonemaster/files/7626827/nsec-bitmaps.pdf)

**Comment by matsduf** [comment]
Zonemaster could send a query to get the NSEC or NSEC3 record of the apex of the zone, and then check that all listed record types in the NSEC/NSEC3 record are present, and none of the non-listed is present. Does that meet your suggestion, @ulrichwisser?

**Comment by matsduf** [comment]
@tgreenx, no, it does not. It is a different issue, and I think it should be a different test case. It can, however, be hard to test.
